agent/internal/config/jwt_test.go

```diff
@@ -0,0 +1,52 @@
+package config
+
+import "testing"
+
+type jwtTest struct {
+	name                string
+	jwtTokenStringified string
+	valid               bool
+}
+
+func (j jwtTest) run(t *testing.T) {
+	_, err := ValidateAndCreateJWT(j.jwtTokenStringified)
+	if err != nil && j.valid {
+		t.Errorf("%v", err)
+	}
+	if err == nil && !j.valid {
+		t.Errorf("expected error")
```

Comment:
If the error is expected, it could simply be t.Log,no?

Reply:
changed accordingly.